spec/back_end/features/confirm_received_request_spec.rb

```diff
@@ -0,0 +1,24 @@
+feature 'confirm received requests' do
+  scenario 'a user can confirm received requests' do
+    sign_up
+    list_test_space
+    click_on "Sign out"
+
+    sign_up_user_2
+    click_on 'BIG HOUSE'
```

Comment:
may need to change the click on BIG HOUSE to be in line with new date logic tests